math/mathcore/inc/Fit/BinData.h

```diff
@@ -109,20 +109,24 @@ public :
    */
    BinData(const BinData & rhs);
 
+   /// assignment operator
    BinData & operator= ( const BinData & rhs );
 
 
    /**
-      preallocate a data set with given size ,  dimension and error type (to get the full point size)
-      If the data set already exists and it is having the compatible point size space for the new points
-      is created in the data sets, while if not compatible the old data are erased and new space of
-      new size is allocated.
-      (i.e if exists initialize is equivalent to a resize( NPoints() + maxpoints)
+      Preallocate a data set with given size ,  dimension and error type.
+      If the data set already exists newPoints are appended to the existing dataset.
+      (i.e if the data exists Initialize is equivalent to a resize( NPoints() + maxpoints).
+      Initialize and Append are equivalent.
```

Comment:
```suggestion
      Preallocate a dataset with given size,  dimension and error type.
      If the dataset already exists, `newPoints` are appended to the existing dataset
      (i.e., if the data exists, Initialize() is equivalent to a `resize( NPoints() + maxpoints )`).
      Initialize() and Append() are equivalent.
```
Id suggest to write `dataset` consistently, improve punctuation and referencing functions such that doxygen can recognize it